kmm/kmm.py

```diff
@@ -18,10 +21,10 @@ def kmm(path: Path):
             "3?",
             "4?",
             "5?",
-            "sweref99_tm_x",
-            "sweref99_tm_y",
+            "x_sweref",
```

Comment:
Same comment as above. Rename longitude/latitude?

Reply:
Do they correspond to longitude/latitude in the same way?

Reply:
Not sure which is which but they work the same way I think